ChatRPG/Pages/Campaign.razor

```diff
@@ -27,9 +32,9 @@
 <div class="container">
     <div class="input-group mx-auto col-8 mt-2 mb-5 input-group-border">
         <input type="text" style="resize:vertical;" class="form-control user-prompt custom-text-field"
-               @bind="_userInput" placeholder="What do you do?" @onkeyup="@EnterKeyHandler"/>
+               @bind="_userInput" placeholder="What do you do?" @onkeyup="@EnterKeyHandler" disabled="@_isWaitingForResponse"/>
         <div class="input-group-append">
-            <button class="btn btn-primary ml-2" type="button" @onclick="SendPrompt">
+            <button class="btn btn-primary ml-2" type="button" @onclick="SendPrompt" disabled="@_isWaitingForResponse">
```

Comment:
Good usability feature my guy.

Reply:
🐝

Reply:
(det er en BeeHappy fra league of legends)